Uplift/Views/Onboarding/CreateProfileView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  CreateProfileView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+import SwiftUI
+import PhotosUI
+
+struct CreateProfileView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @State private var isCheckedTerms: Bool = false
+    @State private var isCheckedData: Bool = false
+    @State private var isCheckedLocation: Bool = false
+    @State private var profileImage: UIImage?
+    @State private var isShowingImagePicker = false
+    @State private var profileItem: PhotosPickerItem?
+
+    // MARK: - UI
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text("Complete your profile.")
+                    .font(Constants.Fonts.h1)
+                    .padding(.leading, 16)
+                Spacer()
+            }
+
+            DividerLine()
+                .upliftShadow(Constants.Shadows.smallLight)
+                .padding(.bottom, 46)
+
+            cameraPlaceholderButton
+                .padding(.bottom, 24)
+
+            Text(mainViewModel.name)
+                .font(Constants.Fonts.h1)
+
+            CheckBoxView(
+                isCheckedTerms: $isCheckedTerms,
+                isCheckedData: $isCheckedData,
+                isCheckedLocation: $isCheckedLocation
+            )
+            .padding(.top, 48)
+
+            if allChecked {
+                Spacer(minLength: 135)
+                readyGreeting
+                getStartedButton
+                    .padding(.top, 32)
+            } else {
+                Spacer(minLength: 200)
+                nextLabel
+            }
+
+            Spacer()
+        }
+        .padding(.vertical, 40)
+        .photosPicker(
+            isPresented: $isShowingImagePicker,
+            selection: $profileItem,
+            matching: .images,
+            photoLibrary: .shared()
+        )
+        .onChange(of: profileItem) { newItem in
+            Task {
+                if let newItem = newItem {
```

Comment:
If you want to just check if an existing value is nil, you can just do:
`if let newItem { ... }`

Also, I would chain all of these into one if-let to keep it clean:

```
if let newItem,
   let data = try? await newItem.loadTransferable(type: Data.self),
   let image = UIImage(data: data) {
              DispatchQueue.main.async {
                  self.profileImage = image
              }
}
```